2016/proposals/narayan-aditya-visualizer.md

```diff
@@ -125,11 +125,11 @@ that the respondent selected.
 
 ![Proposal-1](https://cloud.githubusercontent.com/assets/6830800/13755382/fd10d820-ea40-11e5-8c36-bed3cac6357e.png)
 
-##### PROS:
+**PROS**:
 1. Compatible across all database engines supported by Django.
 2. We can query across the the Foreign Keys between the tables.
 
-#### CONS:
+**CONS**:
```

Comment:
Instead of adding one more "#" you changed the header to bolded text and only in this single proposal.